### ISSUE TITLE
prebuild binaries: Dynamically linked to musl?

### ISSUE THREAD

**Issue by user9931** [zakjan/cert-chain-resolver#25]
I'm trying to run this file  on ubuntu and nothing happens 
ldd says
root@host:/tmp/cert-chain-resolver_linux_amd64$ ldd cert-chain-resolver
        linux-vdso.so.1 (0x00007ffc1537f000)
        libc.musl-x86_64.so.1 => not found

Those builds intended only for alpine linux (using musl)?

**Comment by zakjan** [comment]
Hmm, I thought that Go produces static builds by default. Do you know if there is any flag for `go build` to produce a fully static build? Could you try to compile from sources locally?

**Comment by user9931** [comment]
I'm not a programmer in any means, but `CGO_ENABLED=0 go build ` inside  source's directory works for me
_Thank you for the app_

```
~/apps/cert-chain-resolver/cert-chain-resolver-1.0.3  ldd ./cert-chain-resolver
        not a dynamic executable
```

**Comment by zakjan** [comment]
Hi, thanks for your advice. Could you try the attached build, if it fixes the issue for you?

[cert-chain-resolver_linux_amd64.tar.gz](https://github.com/zakjan/cert-chain-resolver/files/7505457/cert-chain-resolver_linux_amd64.tar.gz)

**Comment by pniederlag** [comment]
Just ran into the very same issues as @thehotery, expecting the release to be executable on my linux machine but experiencing weird "not found" errors until I learned on debugging with ldd and seeing missing library.

the attached build fixes this and is executable

**Comment by zakjan** [comment]
Thanks for confirming, released as 1.0.4.
